app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+from abc import ABC

```

Comment:
Order your imports correctly and add a blank line between different imports groups to follow PEP-8